libs/taskNew.js

```diff
@@ -159,31 +160,80 @@ module.exports = {
         }
     },
 
+    handleImageLinks: (req, res) => {
+        if (req.body.images && req.body.images.length) {
+            const srcPath = path.join("tmp", req.id);
+            async.series(
```

Comment:
can be handled with single appendfile and join.
``fs.appendFIle(`${srcPath}/images.sg`,req.body.files.join('\n'),  (err) => {
    if (err) res.json({ error: err.message });
    else res.json({ success: true });
});``